### PR TITLE
@orta => Removes TestFlight

### LINKED ISSUE #70: Remove TestFlight events
TestFlight have removed FlightPath analytics since Apple bought them, there is no where on the website to view the analytics so I think it's time they are removed from ARAnalytics.

Do we want to remove TestFlight completely from ARAnalytics? Or just remove the events being sent (so you can still `[TestFlight takeOff:identifier]`).

### PR DESCRIPTION
**Pull request by ashfurrow**
This should fix the Travis build issues, too. 

Fixes #70.

### CODE REVIEW

**Comment by orta** [comment]
changelog please

**Comment by ashfurrow** [comment]
RIIIIGHT silly GitHub web interface.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #ARAnalytics
 
+* Removes TestFlight ( @ashfurrow )
```
Comment (orta) [inline CHANGELOG.md]: this is on v3.0.0, I've not been able to release yet, so it will probably be post RAC

Comment (ashfurrow) [inline CHANGELOG.md]: I see. OK!

**Pull request merged**